string from_room
string to_room
---
float32 time
